src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdminService.java

```diff
@@ -207,4 +217,40 @@ public void clearTseAdminDataFromCaseData(CaseData caseData) {
         caseData.setTseAdminSelectPartyNotify(null);
     }
 
+    /**
+     * Creates a pdf copy of the Decision from Tribunal and adds it to the case doc collection.
+     *
+     * @param caseData details of the case from which required fields are extracted
+     * @param userToken autherisation token to use for generating an event document
+     * @param caseTypeId case type to use for generating an event document
+     */
+    public void addTseAdminDecisionPdfToDocCollection(CaseData caseData, String userToken, String caseTypeId) {
+        try {
+            if (isEmpty(caseData.getDocumentCollection())) {
+                caseData.setDocumentCollection(new ArrayList<>());
+            }
+            if (isNullOrEmpty(caseTypeId)) {
+                log.error("Error while creating case document: Case Type ID can not be null or empty");
+                throw new DocumentManagementException(
+                        "Error while creating case document: Case Type ID can not be null or empty");
+            }
+
+            DocumentInfo document = tornadoService.generateEventDocument(caseData, userToken, caseTypeId,
+                    TSE_ADMIN_DECISION_FILE_NAME);
+
+            DocumentTypeItem docItem = createDocumentTypeItem(
+                    documentManagementService.addDocumentToDocumentField(
+                            document),
+                    "Respondent correspondence",
```

Comment:
The Admin Decision shouldn't be tagged as Respondent correspondence, maybe Tribunal correspondence or Judicial direction. Best to check with Dan